src/frontend/demo/src/views/rooms/settings/index.test.tsx

```diff
@@ -1,12 +1,10 @@
-import { render, screen, waitFor } from '@testing-library/react';
+import { buildApiUrl, MagnifyTestingProvider, render } from '@jitsi-magnify/core';
+import { screen, waitFor } from '@testing-library/react';
 import { rest } from 'msw';
 import React from 'react';
-import { createMemoryRouter } from 'react-router-dom';
-import { MagnifyTestingProvider } from '../../../components';
-import { roomConfigMessages } from '../../../components/rooms/RoomConfig/RoomConfig';
-import { defaultRoom } from '../../../mocks/handlers/rooms/roomsHandlers';
+import { createMemoryRouter, RouterProvider } from 'react-router-dom';
+import { defaultRoom } from '../../../mocks/allHandlers/rooms/roomsHandlers';
```

Comment:
```suggestion
import { defaultRoom } from '../../../mocks/handlers/rooms/roomsHandlers';
```

Then move `handlers.ts` within `handlers` folder and rename it `index.ts`